scripts/generate-utilities-docs.js

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env node
+
+const mustache = require("mustache");
```

Comment:
This JS style for this script should be consistent with the rest of this project (single quotes, trailing commas, etc.)

Is it included in the linting?